### PR TITLE
home page setup with bootstrap

### PR DESCRIPTION
**Pull request by sawleha16**
Screenshot from 2022-12-26 21-10-09
Screenshot from 2022-12-26 21-10-12

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -14,3 +14,60 @@
...
+   z-index: 2;
+ }
+
+ .mysignupform input[type="email"] {
+   margin-bottom: -1px;
+ }
+
+ .mysignupform input[type="password"] {
+   margin-bottom: -1px;
+ }
```
Comment (ahmad19) [inline app/assets/stylesheets/application.scss]: i think we do not need this. Just rename `myloginform` to `myform` and use it in both forms

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -1,26 +1,27 @@
-<h2>Log in</h2>
+<div class="text-center">
+  <div class="myloginform w-100 m-auto">
+    <%= image_tag "bootstrap-logo.svg", class: "mb-4", width: "72", height: "57"%>
+      <%= form_for(resource, as: resource_name, url: session_path(resource_name),data: {turbo: false}) do |f| %>
```
Comment (ahmad19) [inline app/views/devise/sessions/new.html.erb]:
```suggestion
      <%= form_for(resource, as: resource_name, url: session_path(resource_name), data: {turbo: false}) do |f| %>
```

File: app/views/devise/sessions/new.html.erb [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
-
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "current-password" %>
-  </div>
+        <div class="form-floating">
+          <%= f.email_field :email, class: "form-control", autofocus: true, autocomplete: "email", placeholder: "name@example.com" %>
+          <%= f.label :email%>
```
Comment (ahmad19) [inline app/views/devise/sessions/new.html.erb]:
```suggestion
          <%= f.label :email %>
```

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (sawleha16) [reply app/assets/stylesheets/application.scss]: done

File: app/views/devise/sessions/new.html.erb (hunk 2 above)
Comment (sawleha16) [reply app/views/devise/sessions/new.html.erb]: Done

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -14,3 +14,60 @@
...
+   z-index: 2;
+ }
+
+ .myform input[type="email"] {
+   margin-bottom: -1px;
+ }
+
+ .myform input[type="password"] {
+   margin-bottom: -1px;
+ }
```
Comment (ahmad19) [inline app/assets/stylesheets/application.scss]: Why do we need it ? you already have the same version written on top

File: app/views/devise/registrations/new.html.erb [hunk 5]
```diff
@@ -1,44 +1,43 @@
...
+    </div>
+
+    <div class="form-floating">
+      <%= f.select(:interests, User::INTEREST_FIELD, { prompt: "Choose interests"}, class: "form-select")%>
+      <%= f.label :interests%>
+    </div>
+      <%= f.submit "Sign up", class: 'w-100 btn btn-lg btn-primary mt-3 mb-3' %>
+      <% end %>
+      <%= render "devise/shared/links" %>
+    </div>
```
Comment (ahmad19) [inline app/views/devise/registrations/new.html.erb]:
```suggestion
    <div class= "myform">
      <%= form_for(resource, as: resource_name, url: registration_path(resource_name), data: {turbo: false}) do |f| %>
      <%= render "devise/shared/error_messages", resource: resource %>

        <div class="form-floating mb-3">
          <%= f.email_field :email, class: "form-control", autofocus: true, autocomplete: "email", placeholder: "name@example.com" %>
          <%= f.label :email %>
        </div>

    <div class="form-floating">
      <%= f.password_field :password, class:"form-control", autocomplete: "new-password", placeholder: "password" %>
      <%= f.label :password %>
        <% if @minimum_password_length %>
          <em>(<%= @minimum_password_length %> characters minimum)</em>
        <% end %>
    </div>

    <div class="form-floating mb-3">
      <%= f.password_field :password_confirmation, class: "form-control", autocomplete: "new-password", placeholder:"password" %>
      <%= f.label :password_confirmation %>
    </div>

    <div class="form-floating mb-3">
      <%= f.text_field :username, class: "form-control", autocomplete: "username", placeholder: "username" %>
      <%= f.label :username %>
    </div>

    <div class="form-floating mb-3">
      <%= f.text_field :age, class: "form-control", autocomplete: "age", placeholder: "age" %>
      <%= f.label :age %>
    </div>

    <div class="form-floating">
      <%= f.select(:interests, User::INTEREST_FIELD, { prompt: "Choose interests"}, class: "form-select")%>
      <%= f.label :interests%>
    </div>
      <%= f.submit "Sign up", class: 'w-100 btn btn-lg btn-primary mt-3 mb-3' %>
      <% end %>
      <%= render "devise/shared/links" %>
    </div>
```

File: app/views/devise/sessions/new.html.erb (hunk 3 above)
Comment (ahmad19) [reply app/views/devise/sessions/new.html.erb]: this is still remaining

**Review by ahmad19 — APPROVED** [review]

**Pull request merged**